python/src/wslink/server.py

```diff
@@ -102,6 +102,12 @@ def add_arguments(parser):
         dest="reverse_url",
         help="Make the server act as a client to connect to a ws relay",
     )
+    parser.add_argument(
```

Comment:
Can you add some comment here on how to use that arg (i.e. example)? As I'm not sure how you pass the certificate and key.